portfolio.html

```diff
@@ -139,6 +139,585 @@ <h2 class="description">I'm a software developer</h2>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="about-section">
+    <div class="about-text">
+      <h1 class="primary-text">About me</h1>
+      <p class="secondary-text .bg-top">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="button">
+        <button type="button">Get My Resume</button>
+      </div>
+    </div>
+    <div class="profession-skills">
```

Comment:
Please fix your margins for this section too. It was part of the requirements for this module as highlighted above

- [ ] This is your code

<img width="448" alt="Screenshot 2022-10-01 at 18 26 58" src="https://user-images.githubusercontent.com/48546992/193416703-af20ecb1-e784-4736-bae1-2ee244c5b1e8.png">

- [ ] This is the figma design
<img width="379" alt="Screenshot 2022-10-01 at 18 27 09" src="https://user-images.githubusercontent.com/48546992/193416736-3208a4f5-e8a6-4d75-b319-e134879214eb.png">